java-generator/maven-plugin/src/main/java/io/fabric8/java/generator/maven/plugin/JavaGeneratorMojo.java

```diff
@@ -30,9 +30,8 @@
     @Parameter(defaultValue = "${project}")
     private MavenProject project;
 
-    // TODO: Modify CRGeneratorRunner to allow passing in a directory too
     @Parameter(property = "fabric8.java-generator.source", required = true)
-    File crdFile;
+    File crdSource;
```

Comment:
maybe we should call this just `source`

And same for `targetDirectory` ---> `target`

Reply:
make sense, let me update the PR.

Reply:
done